src/utils/hooks.js

```diff
@@ -0,0 +1,13 @@
+import { useContext } from "react";
```

Comment:
being picky here but: the `hook.js` file could actually belong to a `context` or `store` folder along with `GlobalState.js` and `appReducer.js`. The hooks are very specific and used only for the context and reducer.

Reply:
Sure thing, I'm all for keeping things organized 👍